CTnlp/patient/__init__.py

```diff
@@ -0,0 +1,76 @@
+from dataclasses import dataclass
+from typing import Union, List, Optional
+
+import defusedxml.ElementTree as ET
+
+from CTnlp.patient.parsers import extract_sections
+from CTnlp.utils import Gender
+
+
+@dataclass
+class Patient:
+    """dataclass containing patient data."""
+
+    unique_id: str
+    patient_id: int
+    description: str
+
+    conditions: Optional[List[str]] = None
+    current_medical_history: Optional[str] = None
+    past_medical_history: Optional[str] = None
+    family_medical_history: Optional[str] = None
+
+    gender: Gender = Gender.unknown
+    age: Union[int, float, None] = None
+    is_healthy: Optional[bool] = None
+    is_smoker: Optional[bool] = None
+    is_drinker: Optional[bool] = None
```

Comment:
wow ,so specific 👌

Reply:
haha, currently this is empty but maybe we can extend it at some point